platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -694,6 +695,17 @@ class Godot(private val context: Context) : SensorEventListener {
 		}
 	}
 
+	fun isDarkModeSupported(): Boolean {
+		return Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q
+	}
+
+	fun isDarkMode(): Boolean {
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+			return context.resources?.configuration?.uiMode?.and(Configuration.UI_MODE_NIGHT_MASK) == Configuration.UI_MODE_NIGHT_YES
+		}
+		return false
+	}
+
```

Comment:
Can you make this methods private, and add the `@Keep` annotation.

Also, add javadocs to describe what they do.